### ISSUE TITLE
builder for Google Compute Engine fails

### ISSUE THREAD

**Issue by alefebvr** [usharesoft/hammr#408]
When trying to build a template with the builder below, the following error is displayed:
```
hammr> template build --file template2.yml
INFO: You provided a yaml file, checking...
Generating 'Google Compute Engine' image (1/1)
ERROR: Builder type unknown: Google Compute Engine
```
I managed to fix the issue by:
- replacing def generate_gce with def generate_google in file hammr/utils/generate_utils.py
- replacing "Google Compute Engine" with "google" in the first type occurrence in the template yaml file.

```
builders:
- type: "Google Compute Engine" 
  account:
    type: "Google Compute Engine"
    name: My GCE Account
    username: my@email.com
    certPassword: XXXXXXX
    cert: "/my/certificate/gce.p12"
  computeZone: europe-west1-a
  bucketLocation: EU
  bucket: mybucket
  projectId: my-project
  storageClass: STANDARD
  diskNamePrefix: uss-
  description: CentOS Core machine image
```

**Comment by alefebvr** [comment]
Note that the hammr documentation will have to be updated accordingly too
See the example on http://docs.usharesoft.com/projects/hammr/en/latest/pages/machine-images/builders/builders-gce.html

**Comment by AliceAlbano** [comment]
Thank you for logging the issue. I was able to reproduce the bug, but for the fix, I only need to update generate_gce by generate_google. The builder type "Google Compute Engine" works for me, so I think the documentation does not need to be updated.

**Comment by AliceAlbano** [comment]
Fixed by #429
